src/main/java/Coach.java

```diff
@@ -0,0 +1,210 @@
+import java.util.Scanner;
+
+public class Coach {
+    static Task[] tasks = new Task[100];
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+
+        handleWelcome();
+
+        while (true) {
+            String input = sc.nextLine().trim();
+            String[] inputs = input.split(" ");
+            String command = inputs[0].toLowerCase();
+
+            switch (command) {
+            case "bye":
+                handleBye();
+                sc.close();
+                return;
+
+            case "list":
+                handleList();
+                break;
+
+            case "mark":
+                handleMark(inputs,true);
+                break;
+
+            case "unmark":
+                handleMark(inputs, false);
+                break;
+
+            case "todo":
+                handleToDo(input);
+                break;
+
+            case "deadline":
+                handleDeadline(input);
+                break;
+
+            case "event":
+                handleEvent(input);
+                break;
+
+            default:
+                printLine();
+                System.out.println("Invalid command!");
+                printLine();
+                break;
+            }
+        }
+    }
+
+    private static void handleToDo(String input) {
+        printLine();
+        String content = input.substring(4);
+        if (content.isEmpty())
+        {
+            printEmpty();
+            return;
+        }
```

Comment:
you can put } and else on the same line 

```suggestion
        } else {
```